circle.yml

```diff
@@ -21,7 +21,7 @@ test:
     - ./lint.sh
     - cd build && make check-format
     - cd build && make check-lint
-    - cd build && make check
+    - cd build && env CTEST_OUTPUT_ON_FAILURE=1 make check
```

Comment:
ho, nice one :+1: 

That being said, it may be better to do it in the CMakefile directly:

```diff
--- a/CMakeLists.txt
+++ b/CMakeLists.txt
@@ -183,7 +183,7 @@ coverage_evaluate()
 
 # Be sure to recompile the test driver if needed before running the test suite.
 add_custom_target(check
-  COMMAND ${CMAKE_CTEST_COMMAND}
+  COMMAND ${CMAKE_CTEST_COMMAND} --output-on-failure
   DEPENDS ${UNIT_TESTS}
   DEPENDS ${EC_DRIVER}
 )
```